src/sumologic-span-processor/find-span-context.ts

```diff
@@ -57,15 +54,17 @@ export const onStart = (span: SdkTraceSpan, context?: Context): void => {
   }
 };
 
-export const onEnd = (span: ReadableSpan): void => {
+export const onEnd = (span: ReadableSpan): boolean => {
```

Comment:
It's not clear to me why a function like `onEnd` should return a boolean and what it means. Comments would definitely help but also maybe the name is not the best?